We can't use geos-config file in conan because it's a non-relocatable file,
therefore not packaged by geos recipe of conan-center-index.
Instead we rely on AutoToolsBuildEnvironment helper to inject proper flags, and
manually inject RTGEOM_GEOS_VERSION.

--- a/configure.ac
+++ b/configure.ac
@@ -84,6 +84,7 @@ AC_CONFIG_FILES([Makefile \
 		headers/librttopo_geom.h \
 		rttopo.pc])
 
+if false; then
 #-----------------------------------------------------------------------
 #   --with-geosconfig
 #
@@ -128,6 +129,7 @@ LIBS="$LIBS $GEOS_LDFLAGS -lgeos_c"
 GEOS_MAJOR_VERSION=`$GEOSCONFIG --version | cut -d. -f1 | sed 's/[[^0-9]]//g'`
 GEOS_MINOR_VERSION=`$GEOSCONFIG --version | cut -d. -f2 | sed 's/[[^0-9]]//g'`
 RTGEOM_GEOS_VERSION="$GEOS_MAJOR_VERSION$GEOS_MINOR_VERSION"
+fi
 AC_DEFINE_UNQUOTED([RTGEOM_GEOS_VERSION], [$RTGEOM_GEOS_VERSION], [GEOS library version])
 AC_SUBST([RTGEOM_GEOS_VERSION])
 
